shared/wallets/transaction/index.js

```diff
@@ -0,0 +1,202 @@
+// @flow
+import * as React from 'react'
+import {Avatar, Box2, Icon, ConnectedUsernames} from '../../common-adapters'
+import Text, {type TextType} from '../../common-adapters/text'
+import {globalColors, globalMargins, platformStyles, styleSheetCreate} from '../../styles'
+// TODO: Format relative dates.
+import {formatTimeForPopup} from '../../util/timestamp'
```

Comment:
Do we have an existing util to display dates relatively, e.g. `Yesterday 8:11 AM` like in the design?

Reply:
I thinner use money to do this

Reply:
Money -> moment

Reply:
Thinner -> think. Damn autocorrect

Reply:
Ah, cool. Fixed, and added a tooltip for the full timestamp